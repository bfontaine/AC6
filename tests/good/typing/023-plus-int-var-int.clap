val x = 42
val y = x + 5

