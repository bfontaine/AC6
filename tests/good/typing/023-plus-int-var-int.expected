:- x = 42
:- y = 47
